readme.md

```diff
@@ -31,32 +33,76 @@ Don't see it? Try reinstalling Package Control. Altenatively, add the repository
 
 If it doesn't work, please file an issue.
 
+If you want contribute or use the latest commits simply git clone the repository `https://github.com/mrjoelkemp/Dependents` to your Packages directory with a directory name of `Dependents`. From sublime choose `Preferences -> Browse Packages`  to easily determine the path to the Packages directory for your install.  Be sure to remove the package via package control first if you had already done so to avoid package control updating the package.
+
+### Configuring settings
+
+There are two ways in which to invoke the package settings.  
+
+One is globally via entires in a user settings file `Preferences -> Package Settings -> Dependents -> Settings-User`.  You can open the `Settings-Default` file and copy and paste the contents as a starter template.   Do not edit the default file as that will be overwritten with updates.
```

Comment:
removed mention of global settings, rewrote a bit of the .deprc description and removed settings menu entry from main.sublime-menu.

Still struggling with the clarity here.   Do you mean for "root" now to refer to js root only?   In which case I would rename "root" to "js_root".   Each language added could have it's own "xxxx_root" parameter would be my suggestion.  I know you started out with js so that is why you used just "root".   How ever this goes this needs to be clearer.  Also I might suggest having changing likewise have a preface js_ for js related settings.  

Once again you started with js but now there is possibility of multi-language support so this would avoid confusion.  As to sass was thinking of a sass_assets, sass_packages or sass_vendor path setting(s) which would solve the issue of when someone does want to look other places for third party sass imports that have no explicit dependency path in the @import like is the case when using bower.

Bottom line is there are still some clarity usage issues with .deprc. This was where I spent the most time trying to get dependents to work correctly.   Take a look at my latest commit first.  For now I just separated js from sass parameters for clarity and added more examples.